solutions/usaco-647.mdx

```diff
@@ -0,0 +1,13 @@
+---
+id: usaco-647
+source: USACO Gold 2016 Open
+title: 248
```

Comment:
the title is being parsed as a number, not sure if that's a problem? @thecodingwizard

Reply:
might be, idk. Just wrap it with quotes
```suggestion
title: '248'
```